functions/command/add.go

```diff
@@ -26,15 +30,38 @@ func NewAddCommand(guildID, activityType, name string) *AddCommand {
 
 func (c *AddCommand) Execute(ctx context.Context, cl *clients.Clients) (*discordgo.WebhookEdit, error) {
 	var typ activity.ActivityType
+	var info *activity.GameInfo
 	switch c.ActivityType {
 	case "activity":
 		typ = activity.ACTIVITY
 	case "game":
-		return utils.NewWebhookEdit("🚧 Games not implemented yet"), nil
+		typ = activity.GAME
+
+		ctx, cancel := context.WithTimeout(context.Background(), time.Duration(time.Millisecond*500))
```

Comment:
Using the existing ctx instead of getting the background one. Is there a reason this needs to complete in 500 ms?

Reply:
it should be fine without it, it was just from the example code